index.html

```diff
@@ -61,5 +61,276 @@ <h4 class="connect">LET'S CONNECT</h4>
+                            <li class="linkedin">
+                                <a
+                                    href="https://www.linkedin.com/in/noel-bryant/"
+                                    rel="noreferrer">
+                                    <img
+                                        src="assets/images/social-media-icons/linkedin.svg" /></a>
+                            </li>
+
+                            <li class="medium">
+                                <a href="index.html" rel="noreferrer">
+                                    <img
+                                        src="assets/images/social-media-icons/medium.svg" />
+
+                                </a>
+                            </li>
+
+                            <li class="github">
+                                <a href="https://github.com/NoelLincoln"
+                                    rel="noreferrer">
+                                    <img
+                                        src="assets/images/social-media-icons/github.svg" />
+                                </a>
+                            </li>
+                            <li class="peacesign">
+                                <a href="index.html">
+                                    <img
+                                        src="assets/images/social-media-icons/peacesign.svg" />
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+
+                    <button class="resume"
+                        onclick="window.location.href='https://drive.google.com/file/d/1iD2S5PJAGP1TI_19C9GD2Zo4FpeSRO1N/view?usp=sharing';">
+                        Get my resume
+                    </button>
+
+                </div>
+                <div class="skill-list">
+                    <ul class="dropdown">
+                        <input type="checkbox" id="dropdown" />
+                        <label for="dropdown-content-languages"
+                            class="dropdown-btn"
+                            id="languages">
+                            <li class="languages">Languages</li>
+                            <span class="arrow south"></span>
+                        </label>
+
+                        <ul class="dropdown-content" role="menu">
+                            <li><img src="assets/images/javascript.png"
+                                    alt="javascript
+                  image" /><a href="#">Javascript</a></li>
+                            <li><img src="assets/images/html.png" alt="html
+                  image" /><a href="#">HTML</a></li>
+                            <li><img src="assets/images/javascript.png"
+                                    alt="javascript
+                  image" /><a href="#">CSS</a></li>
+                        </ul>
+                    </ul>
+                    <ul class="dropdown">
+                        <input type="checkbox" id="dropdown" />
```

Comment:
- [ ] Kindly note that one of the advantages of ul li is for code organization, once you have used ul, the child should be li so this ul should be in li tags each too.

Reply:
the li tags are the children of the unordered list tag, then the list items wrap the anchor tags. Kindly expound